test/fluree/json_ld/impl/iri_test.cljc

```diff
@@ -20,9 +20,9 @@
 
     (is (nil? (iri/parse-prefix "schema::name")))
 
-    (is (nil? (iri/parse-prefix ":schema:name")))
+    (is (= (iri/parse-prefix ":schema:name") [":" "schema:name"]))
```

Comment:
Can you swap the order of these `=` args so that the expectation is first? It'll make test failures easier to read in the future.

Reply:
Addressed in 07173309ce934c33555fab4c1cfe96efc108077b